### PR TITLE
Initial files with the index.js containing the node.js (http2) - version 9.5+ required.

### PR DESCRIPTION
**Pull request by rathishvbr**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var http2 = require('http2');
+
+module.exports = {
+  name: 'ember-http2',
+
+//   init() {
```
Comment (indykish) [inline index.js]: you will need init.

File: config/ember-try.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          }
+        },
+        {
+          name: 'ember-lts-2.18',
+          npm: {
+            devDependencies: {
+              'ember-source': '~2.18.0'
+            }
+          }
+        },
```
Comment (indykish) [inline config/ember-try.js]: Check to see if we need to add 3.0

Remove other (2.12, .16, .18)

File: index.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+//   this._super.init && this._super.init.apply(this, arguments);
+// },
+//
+// included() {
+//   this._super.included.apply(this, arguments);
+//   this.import('vendor/http2.js');
+// },
+//
+// treeForVendor(vendorTree) {
```
Comment (indykish) [inline index.js]: remove this. `treeForVendor`

File: index.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+  this._debugTree = BroccoliDebug.buildDebugCallback('ember-http2');
+},
+
+_shouldCompileJS() {
+  return true;
+},
+
+treeForAddon(tree) {
+  let lodashPath = path.dirname(require.resolve('http2'));
```
Comment (indykish) [inline index.js]: lodashPath = `http2Path`

File: index.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+  let lodashTree = this._debugTree(lodashPath, 'input');
```
Comment (indykish) [inline index.js]: `let http2Tree`

File: index.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+  lodashTree = replace(lodashTree, {
```
Comment (indykish) [inline index.js]: `http2Tree`

File: index.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+      match: /\.js/g,
+      replacement: ''
+    }
+  });
+
+  lodashTree = this._debugTree(lodashTree, 'post-extension-replace');
+
+  lodashTree = new Funnel(lodashTree, {
+    getDestinationPath(path) {
+      // if (path === 'lodash.js') {
```
Comment (indykish) [inline index.js]: What should we do here ?

File: package.json [hunk 8]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "ember-http2",
+  "version": "0.0.1",
+  "description": "The default blueprint for ember-cli addons.",
```
Comment (indykish) [inline package.json]: description: The emberized http2 from node.js *needs node > 9.5

File: package.json [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+  },
+  "repository": "",
+  "scripts": {
+    "build": "ember build",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
+    "test": "ember try:each"
+  },
+  "dependencies": {
+    "ember-cli-babel": "^6.6.0"
```
Comment (indykish) [inline package.json]: `ember-cli-babel` - What dependency do you use ?

File: package.json [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+  },
+  "devDependencies": {
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-ajax": "^3.0.0",
```
Comment (indykish) [inline package.json]: Have you added the dependencies correctly here ? 

The ones you use ?

File: README.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
+ember-http2
+==============================================================================
+
+[Short description of the addon.]
```
Comment (rioosdolphins) [inline README.md]: Emberized http2 version of the node.js (9.5 +) [package-http2](https://nodejs.org/api/http2.html)

File: README.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+
+```
+ember install ember-http2
+```
+
+
+Usage
+------------------------------------------------------------------------------
+
+[Longer description of how to use the addon in apps.]
```
Comment (rioosdolphins) [inline README.md]: The [http2](https://nodejs.org/api/http2.html) package in node.js 9.5+ version has been packaged to be used in ember. 

This has been tested on `ember 3.0.0 +`.

File: package.json [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+    "ember-cli-shims": "^1.2.0",
+    "ember-export-application-global": "^2.0.0",
+    "ember-try": "^0.2.23",
+    "eslint-plugin-ember": "^5.0.0",
+    "eslint-plugin-node": "^5.2.1",
+    "http2": "^3.3.7",
+
+  },
+  "engines": {
+    "node": "^4.5 || 6.* || >= 7.*"
```
Comment (rioosdolphins) [inline package.json]: "node": "^9.5"

**Pull request merged**